### PR TITLE
Added: Avatar component

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added Avatar component.
- Live link --> https://deploy-preview-2--fuzz-ui.netlify.app/pages/docs.html

### CODE REVIEW

File: styles/components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.img-rounded {
+    border-radius: 50%;
+}
```
Comment (DivyanshSareen) [inline styles/components/avatar.css:3]: can use aspect-ratio: 1 / 1 to avoid elliptical images

**Review by DivyanshSareen — APPROVED** [review]
Looks good to merge

File: styles/components/avatar.css (hunk 1 above)
Comment (pratik-kumar-sahu) [reply styles/components/avatar.css:3]: ok this is new, will definitely research on this.

**Review by PrashanthreddyAinala — COMMENTED** [review]
Everything looks good for me

**Pull request merged**